variants/unphone/platformio.ini

```diff
@@ -0,0 +1,17 @@
+[env:unphone]
+;build_type = debug ; to make it possible to step through our jtag debugger 
+extends = esp32s3_base
+board_level = extra
+board = unphone9
+upload_speed = 921600
+monitor_speed = 115200
+monitor_filters = esp32_exception_decoder
+
+build_flags = ${esp32_base.build_flags} 
+  -D PRIVATE_HW
+  -D UNPHONE
```

Comment:
Just a technicality - once this is finished, please register a proper device identifier in our protobufs [here](https://github.com/meshtastic/protobufs/blob/e6b4c590e7c489306c9c44e3ad1fcf62a3efd288/meshtastic/mesh.proto#L219). The PRIVATE_HW is only for development or unpublished ports. You can probably lose the UNPHONE define this way too.

Reply:
Ah yes, I wondered if I should do that now or later.... I'll go and put in a PR to that repo now. Thanks!

Reply:
PR against the protobufs submitted [here](https://github.com/meshtastic/protobufs/pull/477)